netbox/virtualization/models/virtualmachines.py

```diff
@@ -235,6 +242,10 @@ def primary_ip(self):
         else:
             return None
 
+    @property
+    def disk_space(self):
```

Comment:
This should return the value of the `disk` field if no virtual disks have been created.